docs/extending-overseerr/fail2ban.md

```diff
@@ -0,0 +1,93 @@
+# Protecting Your Server Using Fail2ban
+
+{% hint style="warning" %}
+If your OS runs `firewalld`, make sure it is running and that Overseerr is accessible before continuing.
+{% endhint %}
+
+### Configuring Fail2ban
+
+After installing Fail2ban, the configuration files should be located at `/etc/fail2ban`. Make ***copies*** of `jail.conf` and `fail2ban.conf` in the same directoy, replacing the `.conf` extensions with `.local`.
```

Comment:
```suggestion
After installing Fail2ban, the configuration files should be located at `/etc/fail2ban`. Make a copy of `jail.conf` in the same directoy, replacing the `.conf` extensions with `.local`.
```
Just realized that fail2ban.conf is never modified.

Reply:
Yeah, but we have to do that because every time Fail2ban updates, it replaces both `.conf` files, and that could break something. Whatever you change, always change in the .local

Reply:
@costaht Just so you're aware, I use SWAG (which comes with Fail2ban) and there is no `fail2ban.conf` in that directory by default.  I think you should leave it out, since we aren't making any changes anyway.  Also, maybe make a note that for SWAG users, the config files will be located in `/config/fail2ban`?

Reply:
Ok. Can you add that?